### DISCUSSION TITLE
Free licenses for package developers?

### DISCUSSION THREAD

**Discussion by gisostallenberg** [laravel/nova-issues#4911] [Q&A] [answered]
Do you hand out free licenses for package developers?
For example [nova-json](https://github.com/Naoray/nova-json/pull/11#issuecomment-1212420537) is not moving forward because of a missing license. I think the maintainer is not really active anymore, so for him there will be no incentive to purchase a license...

**Comment by holubar** [comment]
I agree. Lot of community developers are leaning towards filament PHP which is bad news. Lot of great plugins are not supported in Nova 4 and they seem to be abandoned.

**Reply by RhysLees** [reply]
Can't say this is a good idea, then everyone would just make crappy packages to claim a free license.
And, how would you even go about it?

We can't just say give me a license and I pinky promise ill make a package.

If the maintainer isn't working on the package anymore, you can always make a fork and update it yourself.

Also, in nova, you can access json values using `->` so there is no need for a json package e.g.

```php
   Text::make('Address', 'data->address_to->address'),
```

**Reply by gisostallenberg** [reply]
@RhysLees nova-json is not about your example.

**Comment by davidhemphill** [comment, marked as answer]
Package maintainers can request a development license via [nova@laravel.com](mailto:nova@laravel.com). We'll consider them on a case-by-case basis.
